stats/influxdb/config.go

```diff
@@ -34,11 +35,13 @@ import (
 
 type Config struct {
 	// Connection.
-	Addr        null.String `json:"addr" envconfig:"INFLUXDB_ADDR"`
-	Username    null.String `json:"username,omitempty" envconfig:"INFLUXDB_USERNAME"`
-	Password    null.String `json:"password,omitempty" envconfig:"INFLUXDB_PASSWORD"`
-	Insecure    null.Bool   `json:"insecure,omitempty" envconfig:"INFLUXDB_INSECURE"`
-	PayloadSize null.Int    `json:"payloadSize,omitempty" envconfig:"INFLUXDB_PAYLOAD_SIZE"`
+	Addr             null.String        `json:"addr" envconfig:"INFLUXDB_ADDR"`
+	Username         null.String        `json:"username,omitempty" envconfig:"INFLUXDB_USERNAME"`
+	Password         null.String        `json:"password,omitempty" envconfig:"INFLUXDB_PASSWORD"`
+	Insecure         null.Bool          `json:"insecure,omitempty" envconfig:"INFLUXDB_INSECURE"`
+	PayloadSize      null.Int           `json:"payloadSize,omitempty" envconfig:"INFLUXDB_PAYLOAD_SIZE"`
+	PushInterval     types.NullDuration `json:"pushInterval,omitempty" envconfig:"INFLUXDB_PUSH_INTERVAL"`
+	ConcurrentWrites null.Int           `json:"concurrentWrites,omitempty" envconfig:"INFLUXDB_CONCURRENT_WRITES"`
```

Comment:
Somewhat unrelated to this PR, but do we know if `omitempty` is correctly handled by `guregu/null`?

[Their README mentions it as a bug](https://github.com/guregu/null#bugs) and there's an [open PR](https://github.com/guregu/null/pull/42) that apparently fixes it.

Reply:
:man_facepalming: yeah... this is problematic, though we probably have bigger issues than the influxdb config :disappointed: I think this might be the result of the same bug: https://github.com/loadimpact/k6/issues/883#issuecomment-481711103

Reply:
It looks like it ... 
maybe after #1070 we should fork the gurugu, merge that PR and use that through the modules ? cause that library looks pretty dead to me ... we can of course first ping the author on whether he will at least review the PR in question